src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -441,7 +441,7 @@ public Guid GetGuid()
         {
             if (!TryGetGuid(out Guid value))
             {
-                throw new FormatException(SR.FormatGuid);
+                throw ThrowHelper.GetFormatException(DataType.Base64String);
```

Comment:
Should this be DataType.Guid?

Reply:
Yes, of course. Thanks for catching that.